pkg/storage/store.go

```diff
@@ -1360,12 +1360,16 @@ func (s *Store) Bootstrap(ident roachpb.StoreIdent, stopper *stop.Stopper) error
 		return err
 	}
 
-	// If we're bootstrapping the store we can campaign idle replicas
-	// immediately. This primarily affects tests.
+	s.Bootstrapped()
+	return nil
+}
+
+// Bootstrapped tells the store that it was bootstrapped and allows the store
+// to campaign idle replicas immediately. This primarily affects tests.
+func (s *Store) Bootstrapped() {
```

Comment:
Maybe call this `NotifyBootstrapped` instead.

s/allows the store to campaign idle replicas/allows idle replicas to campaign/